### ISSUE TITLE
waiting for removal of gulp util from npm packages

### ISSUE THREAD

**Issue by AquiburRKhan** [rehive/dashboard-angular#19]
1) https://github.com/ajwhite/gulp-ng-config/pull/72
2) https://github.com/dlmanning/gulp-sass/pull/647\
4) https://github.com/klei/gulp-angular-filesort/pull/56
5) https://github.com/miickel/gulp-angular-templatecache/pull/151

**Comment by simonua** [comment]
@AquiburRKhan, _gulp-inject_ merged and was released today. I verified in my own implementations that 4.3.1 removes the _gulp-util_ dependency. One less on your list.

**Comment by AquiburRKhan** [comment]
@simonua Thanks a lot for taking time to comment her , really appreciate it :)

**Comment by simonua** [comment]
@AquiburRKhan, you're welcome. I saw this because you referenced an issue or PR of mine for gulp-angular-templatecache. That and gulp-sass are the last outstanding ones in our implementations, and I'm hoping they'll come around. 

Are you sure you wanted to close this issue? There are still outstanding items on your list.

I'm not sure about the state of gulp-ng-config. We use [gulp-ng-constants](https://github.com/guzart/gulp-ng-constant), which appears to do the same, and that has been updated to remove the deprecated _gulp-util_.
